pyomo/contrib/gdpopt/GDPopt.py

```diff
@@ -68,7 +68,9 @@ class GDPoptSolver(object):
         default=600,
         domain=PositiveInt,
         description="Time limit (seconds)",
-        doc="Seconds allowed until terminated"
+        doc="Seconds allowed until terminated. Note that currently only the"
+            "master problem can be checked and terminated, so you might need"
+            "to set a subsolver timelimit too."
```

Comment:
I would prefer the language below, because if the algorithm stalls on the MILP master, we would still have the same issue. (That is, unclear what "master" refers to):

> Seconds allowed until terminated. Note that the time limit can currently only be enforced between subsolver invocations. You may need to set subsolver time limits as well.

Reply:
Fixed in 1aa6815f9d10ba88f7de5ca36cd1bc101b95d0ed